source/gwork/include/Gwork/Controls/CollapsibleList.h

```diff
@@ -61,10 +61,10 @@ namespace Gwk
             {
                 Base::List& children = GetChildren();
 
-                for (Base::List::iterator iter = children.begin(); iter != children.end(); ++iter)
+                for (auto&& iter : children)
```

Comment:
Notice that iter is immedietely assigned into a new variable, after being cast through a gwk_cast.
That variables name is....child.

Everywhere applicable, I used a more descriptive variable name; this just wasn't one of those places.